probatus/stat_tests/distribution_statistics.py

```diff
@@ -377,7 +382,10 @@ def compute(
                     "statistic": statistic,
                     "p_value": p_value,
                 }
-                result_all = result_all.append(result_, ignore_index=True)
```

Comment:
DataFrame.append is deprecated [source](https://pandas.pydata.org/docs/whatsnew/v1.4.0.html#deprecated-dataframe-append-and-series-append) so a better approach would be to first create a list of dictionaries and then transform to DF.

This deprecation warning was adding up with the regular warning generated by Probatus and it was causing the test that checks the number of warnings to fail.

Reply:
Changed :)